doc/nrf/protocols/lte/index.rst

```diff
@@ -0,0 +1,211 @@
+For more information, refer to the `Security protocol for cellular IoT`_ documentation.
+
+
+Getting started with cellular products
+**************************************
+
+For getting started with the cellular products, you can refer to the :ref:`ug_nrf91` documentation, which guides through the features, getting started steps, and advanced usage of the boards.
+For a self-paced, hands-on course, enroll in DevAcademy's `Cellular IoT Fundamentals course`_ , which describes the LTE and NB-IoT technologies and the software architecture of a cellular IoT application based on Nordic Semiconductor’s nRF91 Series device.
+
+The :ref:`nrf9160_ug_network_mode` section provides information about switching between LTE-M and NB-IoT.
+
+Cellular IoT in |NCS|
+*********************
+
+The following sections explain the different applications, libraries, and samples that use the cellular IoT products.
+
+Modem library
+=============
+
+.. include:: ../../device_guides/working_with_nrf/nrf91/nrf91_features.rst
+    :start-after: nrf91_modem_lib_start
+    :end-before: nrf91_modem_lib_end
+
+Library Support
+===============
+
+The following nRF Connect SDK libraries are used for cellular IoT:
+
+* :ref:`lib_modem`
+* :ref:`lib_networking`
+* :ref:`liblwm2m_carrier_readme` library
+* :ref:`mod_memfault` library
+
+Applications and samples
+========================
+
+The following applications use the cellular IoT in |NCS|:
+
+* :ref:`asset_tracker_v2`
+* :ref:`serial_lte_modem`
+* :ref:`connectivity_bridge`
+* `nRF Asset Tracker project`_
+
+The following samples are used for the cellular IoT development in |NCS|:
+
+* :ref:`cellular_samples`
+* :ref:`networking_samples` - These samples use an nRF91 Series device over LTE or an nRF70 Series device over Wi-Fi for communication and connection.
+* :ref:`memfault_sample` sample
+
+Integrations
+============
+
+The following Integrations are available for cellular IoT in |NCS|:
+
+* :ref:`ug_nrf_cloud` - `nRF Cloud`_ is `Nordic Semiconductor's IoT cloud platform`_ that allows you to remotely manage and update your IoT devices using :term:`Firmware Over-the-Air (FOTA) <Firmware Over-the-Air (FOTA) update>`.
+* :ref:`ug_memfault`- A cloud-based web application with |NCS| compatibility that monitors devices and allows you to debug issues remotely through an LTE or Wi-Fi network.
+* :ref:`ug_avsystem` - Software solution provider that provides a device management platform.
+
+Power optimization
+==================
+
+For optimizing the power consumption of your cellular application, you can use the `Online Power Profiler (OPP)`_ tool.
+See :ref:`app_power_opt_nRF91` for more information.
+
+References
+**********
+
+* `nWP044 - Best practices for cellular IoT development`_ - White paper that describes the guidelines that you need to consider when you start the development for a low power cellular IoT product.
+* `Cellular IoT asset tracking`_
+* `Accelerate Cellular Product Development`_
```

Comment:
@martiles : Please confirm are these 2 links required as part of the LTE guide?
* https://youtu.be/b1YW1J4aUps
* https://youtu.be/V-WlGbyRDZg

Please let me know if any other references need to be added.

Reply:
Not required, but nice to have